client/my-sites/site-settings/form-general.jsx

```diff
@@ -379,13 +408,14 @@ class SiteSettingsFormGeneral extends Component {
 	}
 
 	Timezone() {
-		const { fields, isRequestingSettings, siteIsJetpack, translate } = this.props;
-		if ( siteIsJetpack ) {
-			return;
-		}
+		const { fields, isRequestingSettings, translate, supportsLanguageSelection } = this.props;
 
 		return (
-			<FormFieldset>
+			<FormFieldset
+				className={
+					! supportsLanguageSelection && classNames( 'site-settings__has-divider', 'is-top-only' )
```

Comment:
No need to call `classNames()` if we have a static set of classes - we can just use:
```
className={ ! supportsLanguageSelection && 'site-settings__has-divider is-top-only' }
```